client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -69,12 +66,11 @@ export const JetpackSitesDataViews = ( {
 		[]
 	);
 
-	const teams = useSelector( getReaderTeams );
-	const isTeamMember = isAutomatticTeamMember( teams );
+	const isProxied = window.configData?.env_id !== 'a8c-for-agencies-production';
```

Comment:
Hi there! I saw this by confidence.

1. I think you should do `config( 'env_id' )` instead. It will give you that value.
2. Proxied is a concept that shouldn't is not relevant to Calypso and shouldn't be leaked here, maybe we should call this `isNotProduction` or such.

Reply:
That makes sense. Thanks for your quick review @alshakero! [Fixed it here](https://github.com/Automattic/wp-calypso/pull/92065/commits/d1c32622d766aef4d683e859cc7fe80ec0b4b19d).